docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -3842,12 +3773,53 @@ const user = room.getSelf();
     The ID of the User that has been set in the authentication endpoint. Useful
     to get additional information about the connected user.
   </PropertiesListItem>
-  <PropertiesListItem name="isReadOnly" type="boolean" deprecated>
-    Deprecated. False if the user can mutate the Room’s Storage and/or YDoc,
-    true if they can only read but not mutate it.
-  </PropertiesListItem>
 </PropertiesList>
 
+## Deprecated
+
+### Client.enter [@badge=Deprecated]
+
+<Banner title="Deprecated" type="error">
+  This is no longer supported. We recommend using [`client.enterRoom`][]
```

Comment:
@CTNicholas Should we maybe say "This API was removed in 2.0" instead of saying it's not supported?